spec/rspec/matchers/built_in/include_spec.rb

```diff
@@ -51,6 +51,14 @@
           expect("abc\ndef").to include("g", "h")
         }.to fail_matching("expected \"abc\\ndef\" to include \"g\" and \"h\"\nDiff")
       end
+
+      it "does not diff when lines match but are not an exact match" do
+        expect {
+          expect(" foo\nbar\nbazz").to include("foo", "bar", "gaz")
+        }.to raise_error(RSpec::Expectations::ExpectationNotMetError) { |error|
+          expect(error.message).not_to match(/Diff/)
+        }
+      end
```

Comment:
Definitely agree this could (should) be simpler. It's a remnant from when I was trying to examine the diff and the strange characters it contains (I"ve since seen that there are better ways of doing that, too). Do you think it's OK to use `a_string_excluding` here, for the same reason as not using `fail_including` (as indicated at the bottom of this file)?